packages/app/pages/organization/course/[cid]/edit.tsx

```diff
@@ -113,7 +111,8 @@ export default function Edit(): ReactElement {
         sectionGroupNameField === courseData.course.sectionGroupName &&
         zoomLinkField === courseData.course.zoomLink &&
         courseTimezoneField === courseData.course.timezone &&
-        semesterIdField === courseData.course.semesterId &&
+        semesterNameField ===
```

Comment:
Unfortunately, that's how the semester model is structured. We don't want to modify the existing model as it can cause production to break. We would need to eliminate the semester model to make the new version of the model. Given the time, it's easier to accommodate with this approach.

I will probably refactor the model in T2 of the summer.